packages/react/src/components/Table/Table.jsx

```diff
@@ -361,6 +368,23 @@ const propTypes = {
       onRowLoadMore: PropTypes.func,
       /** call back function for when icon button in filter row is clicked  (evt) => {} */
       onFilterRowIconClick: PropTypes.func,
+      /**
+       * Required callback to support drag and drop. This gets the row ID being dragged and the
+       * values for that row. It must return an object of the row IDs that can be dropped on, and a
+       * node use as the preview of what' being dragged (typically the name of the row, possibly
+       * with an icon).
+       *
+       * @type {(rowId: string, values: any[]) => {dropIds: string[], preview: React.Node}}
```

Comment:
some of the other comments/names leads me to think that this is an array vs a string.  Can you drag multiple rows?  or is it just a single row?

Reply:
My first pass was just one row. I since updated it to support multiple rows, so it always passes an array (of 1 or more). This comment (and at least one more) are out of date. I'm pushing a new commit that corrects the comments. Also I've quickly learned to love (and now miss) TypeScript :)